dGame/Entity.cpp

```diff
@@ -317,15 +317,6 @@ void Entity::Initialize() {
 		m_Components.insert(std::make_pair(COMPONENT_TYPE_SOUND_TRIGGER, comp));
 	}
 
-	//Check to see if we have a moving platform component:
```

Comment:
Moved down to after things like ememies and pets get combat and movement AI, so that we can just update their movement AI to have paths and add movement AI to things that only have movement paths and